config/canonical.json

```diff
@@ -35957,4 +36154,4 @@
       "tourism": "motel"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
If you run npm run build you don't end wup with the new line change to the PR. It removes the blank line that most text editors try to add back.

Reply:
I installed nodejs and npm, and added the build-npm-apm extension in atom IDE. I cloned from Github and I ran the "build" function. That IDE eliminates the final space while. And this code is built successfully. Still, I do not know the next step.